src/main/webapp/scripts.js

```diff
@@ -0,0 +1,17 @@
+
+function switchStatus(event, categoryId){
+    event.preventDefault();
+    const xhr = new XMLHttpRequest();
+    xhr.open("POST", "/alteraVisibilidade?id="+categoryId);
+    xhr.addEventListener("load", function (){
+        if(xhr.status===204){
+            var activeStatus = document.querySelector("#active"+categoryId);
+            if(activeStatus.textContent === "true"){
+                activeStatus.textContent = "false";
+            } else {
+                activeStatus.textContent = "true";
```

Comment:
Sobre as aspas https://developer.mozilla.org/pt-BR/docs/Learn/JavaScript/First_steps/Strings#aspas_simples_x_aspas_duplas. Tudo bem usar aspas duplas, pórem se mantenha o uso, não use aspas duplas e simples misturadas, ou uma ou outra. Eu particularmente gosto de usar sempre simples, e em alguns linters também forçam a usar as simples.

Mais outro arquivo para leitura: https://blog.matheuscastiglioni.com.br/string-em-javascript-com-aspas-simples-ou-duplas/

Também acho que seria até melhor usar `Template String`, acho que vocês ainda não viram, mas quando chegar a hora começem a usar, faz muita diferença.

Reply:
Optei por utilizar a aspas simples mesmo, substituí. [dc3a950](https://github.com/gui-lirasilva/CodingSchool/pull/7/commits/dc3a950eaf83b4a454ba32a474e33d541774b135)